<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parkingspacewindow</class>
 <widget class="QDialog" name="parkingspacewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainFrame
{
	alternate-background-color: blue;
	background: darkblue;
	border-radius: 10px;
	border: 2px solid gray;
}

#textFrame
{
	background: qlineargradient(spread:pad, x1:0.38, y1:0, x2:0.722, y2:1, stop:0.6875 #30415D, stop:1 rgba(205, 205, 205, 255));

	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
}

#infoFrame
{
	background: qlineargradient(spread:pad, x1:0.38, y1:0, x2:0.722, y2:1, stop:0.6875 #30415D, stop:1 rgba(205, 205, 205, 255));

	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
}

#titleLabel
{
	color: #fff;
	font: bold 60px;
	font-family: Garamond;
}

#titleLabel2
{
	color: #fff;
	font: bold 60px;
	font-family: Garamond;
}

#titleLabel3
{
	color: #fff;
	font: bold 60px;
	font-family: Garamond;
}

#label
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;
}

#label_2
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;
}

#label_3
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;
}

#startingTime
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;
}

#parkingType
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;;
}

#licenseLabel
{
	color: #fff;
	font: bold 25px;
	font-family: Garamond;
}

#iconLabel
{
	color: rgba(0,0,0,30%);
}

#buttonFrame
{
	background: qlineargradient(spread:pad, x1:0.38, y1:0, x2:0.722, y2:1, stop:0.6875 #30415D, stop:1 rgba(205, 205, 205, 255));

	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
}

#regularButton
{
	color: #fff;
	QGraphicsDropShadowEffect: -2px 2px #346392;
	background-color: #ff9664;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	font-family: Garamond;
	font: bold 45px;

	background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.6875 #438496, stop:1 rgba(205, 205, 205, 155));
}

#regularButton :hover
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,0 5px 15px #193047;
}

#regularButton :active
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,inset 0 5px 30px #193047;
}

#carpoolButton
{
	color: #fff;
	QGraphicsDropShadowEffect: -2px 2px #346392;
	background-color: #ff9664;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	font-family: Garamond;
	font: bold 45px;

	background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.6875 #438496, stop:1 rgba(205, 205, 205, 155));
}

#carpoolButton :hover
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,0 5px 15px #193047;
}

#carpoolButton :active
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,inset 0 5px 30px #193047;
}

#handicapButton
{
	color: #fff;
	QGraphicsDropShadowEffect: -2px 2px #346392;
	background-color: #ff9664;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	font-family: Garamond;
	font: bold 45px;

	background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.6875 #438496, stop:1 rgba(205, 205, 205, 155));
}

#handicapButton :hover
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,0 5px 15px #193047;
}

#handicapButton :active
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,inset 0 5px 30px #193047;
}


#chargingButton
{
	color: #fff;
	QGraphicsDropShadowEffect: -2px 2px #346392;
	background-color: #ff9664;
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d;
	border: 1px solid white;
	border-radius: 15px;
	font-family: Garamond;
	font: bold 45px;

	background: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.6875 #438496, stop:1 rgba(205, 205, 205, 155));
}

#chargingButton :hover
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,0 5px 15px #193047;
}

#chargingButton :active
{
	QGraphicsDropShadowEffect: inset 0 0 0 1px #27496d,inset 0 5px 30px #193047;
}</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="textFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>321</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Parking</string>
     </property>
    </widget>
    <widget class="QLabel" name="titleLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Space Info</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="infoFrame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Starting Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Parking Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>License Plate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="startingTime">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Garamond</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="parkingType">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="licenseLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
